packages/api/src/migrations/0001-add-video-table.ts

```diff
@@ -0,0 +1,12 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20210928102009 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('create table "video" ("id" bigserial primary key, "title" text not null, "durationInSeconds" int not null, "url" text not null);');
+  }
+
+  async down(): Promise<void> {
+    this.addSql('drop table "video";');
+  }
+}
```

Comment:
Okay, I was thinking they weren't party of the story/issue so I removed them.   Just so I understand, every one of our tables will have those fields correct?

Reply:
Yeah 👍

Reply:
In general we shouldn't modify the generated code from `mikro-orm` because that's what is generated from the entities. So if there is something you don't expect or something missing then you should delete the migration and modify the entity until it's generating the code you want. Otherwise every migration going forward will keep making that same change because it thinks it needs that until the Entity is modified.

Reply:
Oh!? Its generating the migration from the entity?  I did the generation first, then modified the sql to be what I wanted, then built the entity.

Reply:
I have re-done the migration creation.